### ISSUE TITLE
Add brew install MAFFT support for pip-based installation

### ISSUE THREAD

**Issue by ericmjl** [modernatx/seqlike#7]
The workflow in `setup.py` should be:

1. Try installing using `dpkg`
2. If fail, try installing using `brew`
3. If fail, raise warning to let user know that they should try to install `mafft` on their own to use the built-in dataframe alignment utilities.

**Comment by aaroncsolomon** [comment]
Was this fixed? I installed seqlike and called align on several aaSeqLike objects but the function failed for lack of installed mafft. I did not see a warning that I should install my own - what was the resolution here? Thanks!

**Comment by ericmjl** [comment]
Hi @aaroncsolomon! Thanks for checking in. 

The tl;dr answer I can provide is to `mamba install -c bioconda mafft`. In my personal experience, nothing has been easier than to get `mamba` to install a package into a `conda` environment. 

The longer answer is that getting `pip` to install `mafft` as part of the dependency chain requires us to make a few assumptions. I think we have `apt` (Debian), and `brew` (Homebrew)-based installations. Others, such as installing from source, are a fairly tall order for us to handle, given the diversity of ways that could fail. 

Not seeing a warning that you should install your own copy of `mafft` does sound like a user-facing bug. Would you be open to raising this as a new issue and contributing to the UX of the package by adding an appropriate warning in a logical place? @andrewgiessel or @ndousis probably have a good sense over where this warning should be placed.
